core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -228,6 +232,38 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
     }
   }
 
+  it should "resolve DOS URIs when submitting workflows" in withDefaultTestDatabase {
```

Comment:
Yeah, it doesn't look so bad, but I feel like I had to go through the 7 stages of grief to come to terms with `withDefaultTestDatabase`.